grails-app/services/org/pih/warehouse/inventory/CycleCountService.groovy

```diff
@@ -380,7 +381,6 @@ class CycleCountService {
         cycleCountItem.countIndex = command.recount ? 1 : 0
         cycleCountItem.status = command.recount ? CycleCountItemStatus.INVESTIGATING : CycleCountItemStatus.COUNTING
         cycleCountItem.dateCounted = new Date()
-        cycleCountItem.assignee = AuthService.currentUser
```

Comment:
this should be in the command too, right? They're allowed to update the assignee in the UI

Reply:
It is in the command

Reply:
oof yeah I somehow missed your above comment